python/sdk/prelude_sdk/models/codes.py

```diff
@@ -182,7 +182,9 @@ class AuditEvent(Enum, metaclass=MissingItem):
     UPDATE_ACCOUNT = 13
     UPDATE_ENDPOINT = 14
     UPDATE_TEST = 15
+    UPDATE_USER = 17
```

Comment:
Shouldn't this come after 16?

Reply:
People (maybe users, mostly devs (ie. me)) scan lists easier if they're in alphabetical order. Since the numbers mean nothing here, I'm prioritizing ordering the names instead of the numbers